log.conf

```diff
@@ -0,0 +1,3 @@
+path = /var/log/opensds
+level = info
+format = [%lvl%]-[%time%]-%msg%
```

Comment:
please move log.conf to opensds.conf in /example/opensds.conf, you code will init log from /etc/opensds/opensds.conf

Reply:
did you address this comment?

Reply:
These config items have been moved to $OPENSDSPATH/example/opensds.conf.